File: GildedRose/python/test_gilded_rose.py
# -*- coding: utf-8 -*-
import unittest
from gilded_rose import Item, update_quality


class GildedRoseTest(unittest.TestCase):
    def test_foo(self):
        items = [Item("foo", 0, 0)]
        update_quality(items)
        self.assertEquals("fixme", items[0].name)


if __name__ == "__main__":
    unittest.main()

File: Matlab/TestGildedRose.m
classdef TestGildedRose < matlab.unittest.TestCase

    methods(Test)
        % Test methods
        function test_standard_item(tc)
            items = [Item("foo", 4, 5)];
            gilded_rose = GildedRose(items);
            gilded_rose.update_quality();
            tc.verifyEqual(items(1).name, "Fixme")
        end
    end
end